.chef/knife.rb

```diff
@@ -0,0 +1,17 @@
+# See http://docs.opscode.com/config_rb_knife.html
+# for more information on knife configuration options
+
+current_dir = File.dirname(__FILE__)
+log_level :info
+log_location STDOUT
+node_name 'fakeclient'
+client_key "#{current_dir}/fakeclient.pem"
```

Comment:
Why does this need to be an empty file? It seems to work for me without any problems as-is.